docs/reference/modules/http.md

```diff
@@ -0,0 +1,186 @@
+
+### `async get(url, options)`
+
+Fetch the given URL (`url`) via a `GET` request and return the response body. If the URL is relative (starts with `/`) it will be requested from the Apostrophe site itself. This is helpful to avoid needing to construct the base URL across environments.
+
+Accepts the following properties on the optional `options` object:
+
+| Option | Type |Description |
+| -------- | ------ | ----------- |
+| `qs` | Object | An object of query parameters and values. |
+| `jar` | Object | Pass in a cookie jar obtained from `apos.http.jar()`. |
+| `parse` | String | Set to `json` to always parse the response body as JSON, otherwise the response body is parsed as JSON only if the `content-type` is `application/json`. |
+| `headers` | Object | An object containing header names and values. |
+| `fullResponse` | Boolean | If `true`, return an object with `status`, `headers` and `body` properties, rather than returning the body directly. The individual `headers` are canonicalized to lowercase names. If a header appears multiple times an array is returned for it. |
+
+If the status code is >= 400 an error is thrown. The error object will be
+similar to a `fullResponse` object, with a `status` property.
+
+### `async head(url, options)`
+
+Make a `HEAD` request for the given URL and return the response object,
+which will include a `status` property as well as `headers`. If the URL is relative (starts with `/`) it will be requested from the Apostrophe site itself. This is helpful to avoid needing to construct the base URL across environments.
+
+Accepts the following properties on the optional `options` object:
+
+| Option | Type |Description |
+| -------- | ------ | ----------- |
+| `qs` | Object | An object of query parameters and values. |
+| `jar` | Object | Pass in a cookie jar obtained from `apos.http.jar()`. |
+| `headers` | Object | An object containing header names and values. |
+
+If the status code is >= 400 an error is thrown. The error object will be
+similar to a `fullResponse` object, with a `status` property.
+
+### `async post(url, options)`
+
+Send a `POST` request to the given URL and return the response body. If the URL is relative (starts with `/`) it will be requested from the Apostrophe site itself. This is helpful to avoid needing to construct the base URL across environments.
+
+Accepts the following properties on the optional `options` object:
+
+| Option | Type |Description |
+| -------- | ------ | ----------- |
+| `qs` | Object | An object of query parameters and values. |
+| `jar` | Object | Pass in a cookie jar obtained from `apos.http.jar()`. |
+| `send` | String | Can be 'json' to always send `options.body` JSON-encoded, or `form` to send it URL-encoded. |
+| `body` | Variable | The request body. If an object or array, it is sent as JSON. Otherwise it is sent as-is, unless the `send` option is set. |
```

Comment:
If it is a `FormData` object constructed via the `form-data` npm module, it is sent as a `multipart/form-data` upload, in order to match the browser-side support for file uploads.